api/src/test/java/org/openmrs/logic/result/EmptyResultTest.java

```diff
@@ -26,4 +26,11 @@ public void isEmpty_shouldReturnTrue() {
 	public void isNull_shouldReturnTrue() {
 		Assert.assertTrue(new EmptyResult().isNull());
 	}
+	@Test
+	public void isNotEmpty_shouldReturnFalse() {Assert.assertFalse(new EmptyResult().exists());
```

Comment:
@Ricky6791 the body of your unit tests is empty and kindly have a look at this https://wiki.openmrs.org/display/docs/Unit+Testing+Conventions